internal/build/iana_rdap.go

```diff
@@ -0,0 +1,49 @@
+package build
+
+import (
+	"encoding/json"
+)
+
+const (
+	ianaRdapURL = "https://data.iana.org/rdap/dns.json"
+)
+
+// FetchRDAPFromIANA retrieves the map of zones to RDAP service endpoints from IANA.
+func FetchRDAPFromIANA(zones map[string]*Zone) error {
+	res, err := Fetch(ianaRdapURL)
+	if err != nil {
+		return err
+	}
+	defer res.Body.Close()
+
+	x := struct {
+		Services [][][]string // I'm screaming at this schema
+	}{}
+	d := json.NewDecoder(res.Body)
+	err = d.Decode(&x)
+	if err != nil {
+		return err
+	}
+	for _, svc := range x.Services {
+		tlds := svc[0]
```

Comment:
I’d call this `domains`, unless RDAP explicitly does *not* support second-level domains.

Reply:
I don't know if RDAP does explicitly support second-level domains or not, @case ?

I'll change to domains though. In my analysis of the downloaded file there were no domains with a . in them, which to my ignorance means they are all TLDs.

Reply:
I'm not sure, but there's [this from CentralNic](https://centralnicregistry.com/support/information/rdap):

> Unfortunately, since the IANA can only accept registrations for top-level domains, CentralNic's SLD portfolio cannot be added to the registry. However, the RDAP service is available for these domains, at https://rdap.centralnic.com/{sld}

e.g. https://rdap.centralnic.com/us.com/domain/hello.us.com

Reply:
Okay thanks Eric, so I don't think we need to make any changes here.